src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/index.tsx

```diff
@@ -53,24 +59,45 @@ const SearchTabPanel = () => {
             setQueryString,
         } = useQueryStore.getState();
 
+        const {kqlFilterInput} = useViewStore.getState();
+
         setQueryIsCaseSensitive(queryIsCaseSensitive);
         setQueryIsRegex(queryIsRegex);
         setQueryString(queryString);
 
         copyPermalinkToClipboard({}, {
             logEventNum: URL_HASH_PARAMS_DEFAULT.logEventNum,
-            queryString: queryString,
+            query: kqlFilterInput,
             queryIsCaseSensitive: queryIsCaseSensitive,
             queryIsRegex: queryIsRegex,
+            subquery: queryString,
         });
     }, []);
 
+    const isKqlFilteringEnabled = null !== fileTypeInfo &&
+        "CLP IR" === fileTypeInfo.name &&
+        true === fileTypeInfo.isStructured;
+
     return (
         <CustomTabPanel
             tabName={TAB_NAME.SEARCH}
             title={TAB_DISPLAY_NAMES[TAB_NAME.SEARCH]}
             titleButtons={
                 <>
+                    {isKqlFilteringEnabled &&
+                    <PanelTitleButton
+                        title={"What is KQL log filter?"}
+                    >
+                        <Link
+                            color={"neutral"}
+                            href={"https://docs.yscope.com/yscope-log-viewer/main/user-guide/struct-logs/format/index.html"}
```

Comment:
maybe this is the intended link?
```suggestion
                            href={"https://docs.yscope.com/clp/main/user-docs/reference-json-search-syntax.html"}
```